lib/puppet/application/filebucket.rb

```diff
@@ -4,6 +4,8 @@ class Puppet::Application::Filebucket < Puppet::Application
 
   option("--bucket BUCKET","-b")
   option("--debug","-d")
+  option("--fromdate FROMDATE","-f")
+  option("--todate TODATE","-t")
```

Comment:
Would it make sense to add a `--path` (list only) option that lists entries in the file buck for files that contain the given value in their path?